### PR TITLE
fix: add style improvements

### PR DESCRIPTION
**Pull request by acezard**
Font size, font family, keyboard behaviour

### CODE REVIEW

File: src/screens/lock/LockScreen.tsx [hunk 1]
```diff
@@ -131,6 +134,15 @@ export const LockScreen = (props: LockScreenProps): JSX.Element => (
   <>
     <FlagshipBars />
 
-    <LockView {...useLockScreenProps(props.route?.params)} />
+    <TouchableWithoutFeedback
+      onPress={Keyboard.dismiss}
```
Comment (Ldoppea) [inline src/screens/lock/LockScreen.tsx:138]: This dismiss the keyboard, but does it remove the focus on text field too?

Comment (acezard) [reply src/screens/lock/LockScreen.tsx:138]: it does indeed

**Comment by Crash--** [comment]
No safe area for the button?

**Comment by Ldoppea** [comment]
@acezard I force pushed a small fix.

This import does not work (at least on iOS side) 
```js
import { TouchableWithoutFeedback } from 'react-native-gesture-handler'
```
The behavior is that the `<TouchableWithoutFeedback>` wrapper makes it impossible to click on any button of the screen (only the text input is still interactive)

But if instead we use
```js
import { TouchableWithoutFeedback } from 'react-native'
```

Then we have the correct behavior. It seems like `react-native-gesture-handler` is an old dependency that should be updated or maybe removed. I'll investigate on it.

**Review by Crash-- — APPROVED** [review]

**Pull request merged**